examples/sdk-app-example/app/src/main/java/com/example/bugsnag/android/ExampleApplication.kt

```diff
@@ -22,12 +22,22 @@ class ExampleApplication : Application() {
 
     private external fun performNativeBugsnagSetup()
 
+    private val multiProcessApp = true
```

Comment:
I don't think we really need this, but happy to tidy this up nearer the end of the project when we uncomment the `persistenceDirectory`.